### PR TITLE
sarek blog post

### PR DESCRIPTION
**Pull request by FriederikeHanssen**
@netlify /blog/2024/sarek_paper

### CODE REVIEW

File: src/content/blog/2024/sarek_paper.mdx [hunk 1]
```diff
@@ -0,0 +1,75 @@
+---
+title: 'nf-core/sarek paper'
+subtitle: 'Harder, Better, Faster, Stronger'
+pubDate: 2024-04-25T00:13:00+01:00
+
+headerImage: '/assets/images/blog/sarek/skierfe.jpg'
+headerImageAlt: 'Skierfe'
```
Comment (mashehu) [inline src/content/blog/2024/sarek_paper.mdx:7]: That's not a good description for people not seeing the image 😉

File: src/content/blog/2024/sarek_paper.mdx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+authors:
+    - 'FriederikeHanssen'
+    - 'maxulysse'
+label:
+    - 'community'
```
Comment (mashehu) [inline src/content/blog/2024/sarek_paper.mdx:13]:
```suggestion
    - 'pipeline'
```

File: src/content/blog/2024/sarek_paper.mdx [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+import { Image } from 'astro:assets';
+
+We are extremely happy to see this [paper](https://academic.oup.com/nargab/article/6/2/lqae031/7658070) out describing the changes and
+updates to nf-core/sarek, the DNA variant calling pipeline, in the last several years.
+
+In 2020 we embarked on the journey of rewriting the whole pipeline in DSL2. One of the major motivations was to bring down
+cloud computing costs and generally reduce storage space and computational resources. Overall, we reduced computational
+costs on AWS (last summer, using spot instances) by 70% to about $20 from FASTQs to annotated VCFs using Strelka, Manta, and VEP.
+
+But first an overview - no nf-core pipeline without a metro map :
```
Comment (mashehu) [inline src/content/blog/2024/sarek_paper.mdx:36]:
```suggestion
But first an overview - no nf-core pipeline without a metro map:
```

File: src/content/blog/2024/sarek_paper.mdx [hunk 4]
```diff
@@ -0,0 +1,77 @@
+---
+title: 'nf-core/sarek paper'
+subtitle: 'Harder, Better, Faster, Stronger'
+pubDate: 2024-04-25T00:13:00+01:00
+
+headerImage: '/assets/images/blog/sarek/skierfe.jpg'
+headerImageAlt: 'Skierfe'
```
Comment (maxulysse) [inline src/content/blog/2024/sarek_paper.mdx:7]:
```suggestion
headerImageAlt: 'Mount Skierfe outlined showing the inspiration for the Sarek logo'
```

File: src/content/blog/2024/sarek_paper.mdx [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+But first an overview - no nf-core pipeline without a metro map :
+
+<Image src={sarek_subway} alt="Metromap of nf-core/sarek" height="300" />
+
+We ditched BAM format where possible and switched to CRAM saving us 4x work storage space.
+
+<Image src={cram_meme} alt="Toy story meme" height="300" />
+<Image src={cram_graph} alt="Bar chart showing reduction in storage space usage" height="300" />
+
+The `s`plitFastq()` operator was replaced by a FastP process to split the Fastq
```
Comment (maxulysse) [inline src/content/blog/2024/sarek_paper.mdx:45]:
```suggestion
The `splitFastq()` operator was replaced by a FastP process to split the Fastq
```

File: src/content/blog/2024/sarek_paper.mdx [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+files before alignment (default 12), and changed the default grouping of the intervals for BQSR to 21 reducing storage space another 4x
```
Comment (maxulysse) [inline src/content/blog/2024/sarek_paper.mdx:46]:
```suggestion
files before alignment (default 12) plus replacing trimgalore! for read trimming, and changed the default grouping of the intervals for BQSR to 21 reducing storage space another 4x
```

**Comment by FriederikeHanssen** [comment]
@mashehu the metro map looks kinda small. can we increase the size?

**Comment by FriederikeHanssen** [comment]
@nf-core-bot fix linting

**Review by maxulysse — APPROVED** [review]

File: src/content/blog/2024/sarek_paper.mdx [hunk 7]
```diff
@@ -0,0 +1,76 @@
+---
+title: 'nf-core/sarek paper'
+subtitle: 'Harder, Better, Faster, Stronger'
+pubDate: 2024-04-26T00:13:00+01:00
```
Comment (maxulysse) [inline src/content/blog/2024/sarek_paper.mdx:4]: shouldn't the date be today's?

Comment (FriederikeHanssen) [reply src/content/blog/2024/sarek_paper.mdx:4]: yeah but when I put date today it put april 24ths on the preview 🤨

**Review by mashehu — APPROVED** [review]

**Review by maxulysse — APPROVED** [review]

**Pull request merged**